jsapp/js/components/common/button.tsx

```diff
@@ -141,12 +141,25 @@ const Button = (props: ButtonProps) => {
     additionalButtonAttributes['data-cy'] = props['data-cy'];
   }
 
+  const handleClick = (event: React.BaseSyntheticEvent) => {
+    if (!props.isDisabled && props.onClick) {
+      props.onClick(event);
+    }
+  };
+
+  const onKeyUp = (event: React.KeyboardEvent<HTMLButtonElement>) => {
+    if (event?.key === 'space' || event?.key === 'enter') {
+      handleClick(event);
+    }
+  };
+
   return (
     <button
       className={classNames.join(' ')}
       type={props.isSubmit ? 'submit' : 'button'}
-      disabled={props.isDisabled}
+      aria-disabled={props.isDisabled}
```

Comment:
This change allows me to tab-navigate into a disabled button and activate it with spacebar/enter. You can test it at `#/forms/<uid>/settings/media`. My guess would be that `onClick={props.onClick}` has to be changed into `onClick={handleClick}` to fix that.

Reply:
Just a thought: it would be good to announce on Zulip that this is our new direction as for disabled UI elements - they should be tab-navigable with visible focus, just `0.5` opacity and not activable. We could reference https://ux.stackexchange.com/questions/103239/should-disabled-elements-be-focusable-for-accessibility-purposes